include/zephyr/bluetooth/audio/audio.h

```diff
@@ -398,16 +398,16 @@ enum bt_audio_dir {
  *  @param _latency Maximum Transport Latency (msec)
  *  @param _pd Presentation Delay (usec)
  */
-#define BT_AUDIO_CODEC_QOS(_interval, _framing, _phy, _sdu, _rtn, _latency, _pd) \
-	{ \
-		.interval = _interval, \
-		.framing = _framing, \
-		.phy = _phy, \
-		.sdu = _sdu, \
-		.rtn = _rtn, \
-		.latency = _latency, \
-		.pd = _pd, \
-	}
+#define BT_AUDIO_CODEC_QOS(_interval, _framing, _phy, _sdu, _rtn, _latency, _pd)                   \
+	((struct bt_audio_codec_qos){                                                              \
```

Comment:
```suggestion
	((struct bt_audio_codec_qos) {                                                             \
```

Reply:
In Zephyr we do not have spaces between the cast type and the value. See e.g. 
```c
((struct bt_audio_codec_cfg){
((struct bt_audio_codec_cap){
(struct reg_addr){1, 36}
```

etc. although that is not the case in `bluetooth.h` where we have
```c
((struct bt_le_ext_adv_start_param[]) {
((struct bt_le_scan_param[]) {
BT_DATA(_type, ((uint8_t []) { _bytes }), \
```

etc.

I suggest to keep as is, as I believe that is considered the norm in Zephyr, but we do not have either of the ways documented as the "right" way anywhere.